src/app/pages/card-import/card-import.page.ts

```diff
@@ -129,7 +140,7 @@ export class CardImportPage implements OnInit {
       };
 
       this.cardService.addCard(this.card).then(() => {
-         this.router.navigate(["/profile"], navigationExtras);
+         this.router.navigate(["/home/profile"], navigationExtras);
```

Comment:
It is likely that this page does not know about /home/profile. Simply using /profile which is available globally in app-routing.module.ts works just as well. Focus on restructuring the /home sub-route later as a new issue.